drivers/sensor/ti/tmag5273/tmag5273.c

```diff
@@ -398,10 +398,11 @@ static inline uint8_t tmag5273_get_fetch_block_size(const struct tmag5273_config
 /** @brief returns the size of the CRC field if active */
 static inline uint8_t tmag5273_get_crc_size(const struct tmag5273_config *drv_cfg)
 {
+#if CONFIG_CRC
 	if (drv_cfg->crc_enabled) {
 		return TMAG5273_CRC_I2C_SIZE;
 	}
-
+#endif
```

Comment:
Prefer `ifdef` and add `ARG_UNUSED(drv_cfg)` in the `#else` case.

Reply:
done

Also changed this for all other occurrences.